.chloggen/unexport_OTelRecordDatastream.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
Personally I wouldn't consider this as a breaking change as we're unexporting structs from a test file. I don't think this needs a changelog at all, but that's up for debate.

Reply:
agree, I will remove changelog